zds/tutorialv2/managers.py

```diff
@@ -26,7 +26,7 @@ def __get_list(self, subcategories=None, tags=None, content_type=None, with_comm
         if content_type is not None:
             if not isinstance(content_type, list):
                 content_type = [content_type]
-
+            content_type = list(filter(None, content_type))
```

Comment:
pareil, la matérialisation en `list` est inutile (puisqu'on fait un for direct après dessus) et néfaste niveau perfs.